modules/099-ceph-csi/crds/cephcsi.yaml

```diff
@@ -0,0 +1,131 @@
+metadata:
+  name: cephcsi.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: ceph-csi
+    app: ceph-csi
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: cephcsi
+    kind: CephCSI
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema: &schema
+        openAPIV3Schema:
+          type: object
+          description: |
+            Ceph cluster connection parameters and StorageClasses configuration
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - clusterID
+                - userID
+                - userKey
+                - monitors
+              properties:
+                clusterID:
+                  description: |
+                    Ceph FSID/UUID
+                  type: string
+                userID:
+                  description: |
+                    Username without `client.`
+                  type: string
+                userKey:
+                  description: |
+                    Ceph auth key corresponding to userID
+                  type: string
+                monitors:
+                  description: |
+                    List of ceph-mon IP addresses in the format `10.0.0.10:6789`
+                  type: array
+                  items:
+                    type: string
+                rbd:
+                  type: object
+                  properties:
+                    storageClasses:
+                      description: |
+                        Description of StorageClasses for Rados Block Device (RBD)
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          name:
+                            description: |
+                              StorageClass name
+                            type: string
+                          pool:
+                            description: |
+                              Pool name for creating RBD images
+                            type: string
+                          reclaimPolicy:
+                            description: |
+                              The reclaim policy for a PersistentVolume
+                              https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming
+                            type: string
+                          allowVolumeExpansion:
+                            description: |
+                              Allows the users to resize the volume by editing the corresponding PVC object
+                            type: boolean
+                          mountOptions:
+                            description: |
+                              List of mount options
+                            type: array
+                            items:
+                              type: string
```

Comment:
Do we need to validate these options?

Reply:
I don't see how we can validate this. It's just a list of mount options.

Reply:
I checked, and it seems Kubernetes also skips validation for mountOptions, and so will we.